components/Newsletter.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import classnames from 'classnames'
+
+interface NewsletterFormProps {
+  wrapperClassName?: string
+  className?: string
+}
+const NewsletterForm: React.FC<NewsletterFormProps> = ({ wrapperClassName = '', className = '' }) => { 
```

Comment:
This component is going to be presented in two scenarios:
1. In the newsletter page with while background and black letters. (default)
2. In the landing page call to action banners, with blue background and white letters. (see image)

![Screenshot 2024-02-21 at 11 45 31](https://github.com/json-schema-org/website/assets/40007659/286c2123-7a19-4ba9-bc1e-d37541098c39)

With that in mind you will need to define an input parameter allowing to define what of those 2 behaviours the component will render.